### ISSUE TITLE
Story too short.

### ISSUE THREAD

**Issue by kqwyf** [4th-year-in-tsinghua/4th-year-in-tsinghua#1]
**Description:** The story is too short, which make the readers look forward to the following up plot.

**Steps to reproduce:**
1. Open `main.pdf` and get a cup of milk.
2. Read it.
3. After reading, drink the milk.
4. Say: "啊，好期待下一章啊。"

**Comment by zdy023** [comment]
You're right. It's too short. But I have no ability to compose a long long story. I'm so weak as you know.

**Comment by Randool** [comment]
Hasn't it been updated for a long time?

**Comment by zdy023** [comment]
No, it hasn't. I haven’t got time enough for this work after arriving at the campus.

**Comment by kqwyf** [comment]
> Yes, it is. I haven’t got time enough for this work after arriving at the campus.

`Yes, it is.` -> `No, it hasn't.`

Weak man?

**Comment by zdy023** [comment]
ok, ok, you're right. I have just corrected it.

**Comment by Timothyxxx** [comment]
Agree, the story is too short.
Consider to propose a new one about HKU and Hong Kong
